src/mumble/AudioInput.cpp

```diff
@@ -815,13 +816,30 @@ void AudioInput::encodeAudioFrame() {
 
```

Comment:
Can we change this code to always keep `activityState` up-to-date?

I think it would be something like: